process/incubation.md

```diff
@@ -0,0 +1,138 @@
+
+#### Sandbox Level
+
+To be accepted at the Sandbox level, a project must meet the [SSWG minimal requirements](#minimal-requirements) detailed below and be endorsed by at least two SSWG sponsors.
+
+Early adopters should treat early stage projects with extra care. While Sandbox projects are safe to try out, it is expected that some projects may fail and never move to the next maturity level. There is no guarantee of production readiness, users, or professional level support. As such, users must exercise their own judgment.
+
+#### Incubating Level
+
+To be accepted at Incubating level, a project must meet the Sandbox level requirements plus:
+
+* Document that it is being used successfully in production by at least three independent end users which, in the SSWG judgement, are of adequate quality and scope.
+* Have a healthy number of committers.
+* Demonstrate a substantial ongoing flow of commits and merged contributions.
+* Receive a supermajority vote from the SSWG to move to Incubation stage.
+
+#### Graduated Level
+
+To be accepted at Graduated level, a project must meet the Incubating level criteria plus:
+
+* Adopt all [SSWG best practices](#best-practices), as detailed below.
+* Have committers from at least two organizations.
+* Receive a supermajority vote from the SSWG to move to Graduation stage.
+
+### Ecosystem Index
+
+All projects and their respective levels will be listsed on the Swift Server Ecosystem index page. In cases where more than one project solves a particular problem (e.g., two similar database drivers), they will be ordered by popularity. The SSWG reserves the right to define a singular solution for critical building blocks, such as Logging or Metrics APIs, where consistency across the ecosystem is of a critical nature.
+
+The SSWG will meet every 6 month to review all projects, and it reserves the right to demote, archive, or remove projects that no longer fulfill minimal requirements. For example, a Graduated project that no longer receives regular updates or fails to address security concerns in timely fashion. Similarly, the SSWG reserves the right to remove or archive Pitches and Proposals that no longer receive updates.
+
+Changes to the Swift Server Ecosystem index page will be announced by the SSWG using the Swift Server forums.
+
+## Minimal Requirements
+
+* General
+  * Publicly accessible source managed by an SCM such as github.com or similar
+  * Adopt the [Swift Code of Conduct](https://swift.org/community/#code-of-conduct)
+  * [Developer Certificate of Origin](https://developercertificate.org) or a [Contributor License Agreement](https://en.wikipedia.org/wiki/Contributor_License_Agreement)
```

Comment:
IIUC, Swift itself neither has a Developer Certificate of Origin or a CLA

Reply:
IBM does require a CLA for contributions to our projects, but as Swift.org and Vapor don't I don't think we should require one here.

Reply:
Let's discuss offline the CLA aspect.  It's a potential barrier to entry.

Reply:
BTW, we can leave the CLA part out for now, and come back to it during the RFC discussion with the general community.

Reply:
... or we can label this as a "discussion point" for which we want the public to provide commentary.